library/core/class.model.php

```diff
@@ -510,7 +510,7 @@ public function delete($where = [], $options = []) {
             deprecated('The $limit parameter is deprecated in Gdn_Model->delete(). Use the limit option.');
             $limit = $options;
         } else {
-            $options += ['rest' => true, 'limit' => null];
+            $options += ['reset' => true, 'limit' => null];
```

Comment:
This is one of those things that requires special attention. When you combine this bug with [this one](https://github.com/vanilla/vanilla/pull/10879/files#diff-cc39bda16852db3b521fa7c8dd7f6cdeL1396) you'll see the situation is a bit delicate. I think we have a decent number of tests that delete and would fail if we all of a sudden stopped resetting the `Gdn_SQLDriver` though.